web/src/message_list.js

```diff
@@ -426,8 +426,8 @@ export class MessageList {
         $recipient_row.find(".on_hover_topic_unmute").show();
     }
 
-    reselect_selected_id() {
-        const selected_id = this.data.selected_id();
+    reselect_selected_id(selected_id = undefined) {
+        selected_id = selected_id ?? this.data.selected_id();
```

Comment:
Yeah, did that.

Reply:
OK -- can you explain in the commit message precisely why it's possible for the IDs to be different? Or maybe better would be a comment of the form:

// We can't use reselect_selected_id here, because ...

Reply:
My bad, it looks like we can just directly call `reselect_selected_id` here. I was think we needed for local echo messages but it looks like `reify_message_id` is called before this call, so we don't need to take care of that.